cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -626,4 +633,166 @@ define([
             });
         });
     });
+
+    describe('GroupList', function() {
```

Comment:
Sure, I agree. Though I noticed a pattern when naming these classes - all of the group configuration related view classes did not use the word "View" in them, while I'm used to seeing Backbone view class names end in "View". Which pattern do we prefer? (I accept that appending "View" in the test description makes things easier to read).